core.coffee

```
@@ -375,8 +375,11 @@
 						throw new Error('Either the url or resource must be specified.')
 					url = params.resource
 
-					if params.id?
-						url += '(' + escapeValue(params.id) + ')'
+					if params.hasOwnProperty('id')
+						if params.id?
```

nit, but I'd go the fail early way:

```
if params.hasOwnProperty('id')
  if not params.id
    throw new Error('...')
  url += '(' + escapeValue(params.id) + ')'
```